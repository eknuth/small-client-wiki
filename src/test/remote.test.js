import React from 'react'
import renderer from 'react-test-renderer'

import Remote from '../remote'

jest.mock('../page', () => 'Page')

describe('Remote', () => {
  const instance = {
    "context": undefined,
    "id": 44000000000,
    "site": "ward.bay.wiki.org",
    "slug": "active-journal",
    "story": undefined,
    "title": undefined
  }
  it('renders correctly', () => {
    expect(renderRemote({instance})).toMatchSnapshot()
  })
})

function renderRemote(props = {}) {
  return renderer.create(
    <Remote {...props}></Remote>
  ).toJSON()
}