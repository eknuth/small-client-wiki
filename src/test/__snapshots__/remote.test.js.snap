exports[`Remote renders correctly 1`] = `
<Page
  instance={
    Object {
      "context": undefined,
      "id": 44000000000,
      "site": "ward.bay.wiki.org",
      "slug": "active-journal",
      "story": undefined,
      "title": undefined,
    }
  } />
`;
